lib/runner.js

```diff
@@ -46,6 +46,8 @@ util.inherits(Runner, EventEmitter);
 module.exports = Runner;
 
 optionChain(chainableMethods, function (opts, title, fn) {
+	fn = fn || function () {};
```

Comment:
Cool, can do. Was also wondering if `||` is enough, or should I explicitly be checking for `typeof fn === 'undefined'`?